src/execution/syscalls.rs

```diff
@@ -107,34 +108,33 @@ pub fn call_contract(
 //     Ok(DeployResponse { contract_address: deployed_contract_address, constructor_retdata })
 // }
 
-// TODO: EmitEvent syscall.
 // #[derive(Debug, Eq, PartialEq)]
 // pub struct EmitEventRequest {
-//     pub content: EventContent,
+//     pub keys: Vec<Felt252>,
+//     pub data: Vec<Felt252>,
```

Comment:
Why did you change commented code?

Reply:
It was previous version of `EmitEventRequest`. Removed.